analyzers/src/SonarAnalyzer.CSharp/Rules/DisposableNotDisposed.cs

```diff
@@ -224,7 +241,7 @@ private static void ExcludeReturnedPassedAndAliasedLocalsAndPrivateFields(Syntax
                     throw new NotSupportedException("Syntax node should be either an identifier or a simple member access expression");
                 }
 
-                if (IsStandaloneExpression(expression)
+                if ((IsStandaloneExpression(expression) || IsConfigureAwaitInvocationInsideUsingStatement(expression))
```

Comment:
[education] why is `IsStandaloneExpression` important for exclusion? 

[for this PR] semantically, shouldn't this `IsConfigureAwaitInvocationInsideUsingStatement` be inside `ExcludeDisposedAndClosedLocalsAndPrivateFields` rather than inside `ExcludeReturnedPassedAndAliasedLocalsAndPrivateFields`?

the `await using` is actually doing the disposal, right?

Reply:
I simplified it by only checking for the using statement and not specifically for the `ConfigureAwait `method.

Reply:
nice!